packages/foundation-ui/src/mount/__tests__/Mount-test.js

```diff
@@ -36,14 +36,14 @@ describe("Mount", function() {
     ).toBeDefined();
   });
 
-  it("should render null if no component is registered and no children defined", function() {
+  it("render null if no component is registered and no children defined", function() {
     const renderer = TestUtils.createRenderer();
     renderer.render(<Mount type="children-test" />);
 
     expect(renderer.getRenderOutput()).toBe(null);
   });
 
-  it("should not wrap a single child", function() {
+  it("not wrap a single child", function() {
```

Comment:
Maybe we should rename the "should not...", to "do not..." or don't

Reply:
"does not" 👍